demos/object_detection_demo/cpp/main.cpp

```diff
@@ -265,8 +265,6 @@ cv::Mat renderDetectionData(DetectionResult& result, const ColorPalette& palette
         std::ostringstream conf;
         conf << ":" << std::fixed << std::setprecision(1) << obj.confidence * 100 << '%';
         const auto& color = palette[obj.labelID];
-        cv::putText(outputImg, obj.label + conf.str(),
-            cv::Point2f(obj.x, obj.y - 5), cv::FONT_HERSHEY_COMPLEX_SMALL, 1, { 230, 230, 230 }, 3);
         cv::putText(outputImg, obj.label + conf.str(),
             cv::Point2f(obj.x, obj.y - 5), cv::FONT_HERSHEY_COMPLEX_SMALL, 1, color);
```

Comment:
```suggestion
        putHighlightedText(outputImg, obj.label + conf.str(),
            cv::Point2f(obj.x, obj.y - 5), cv::FONT_HERSHEY_COMPLEX_SMALL, 1, color, 2);
```
it's better to use `putHighlightedText` fucntion for labels.

Reply:
btw, it wasn't a double print you fixed, it was the way to highlight text before `putHighlightedText` was added, Just in object detection it wasn't updated for some reason.

Reply:
ok, committed suggestion